docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+Geotrellis Quickstart Guide
+===========================
+
+Geotrellis is a large and evolving package, which makes it difficult to maintain current documentation.  To make up for this deficiency, this quickstart guide will provide a set of documents describing case studies that accomplish some simple goals.  These examples are meant to highlight the fundamental Geotrellis classes as well as show some basic Scala techniques for those new to the language.
+
+Getting Started
+---------------
+
+In the following, we'll be providing some Scala code samples that make use of the GeoTrellis library.  This requires having a running Scala instance with GeoTrellis set as a library dependency.  It is recommended that the user download and install SBT, the Scala Build Tool.  Once that is installed, it is recommended that you use `git` to clone the repository at `github.com/geotrellis/geotrellis-sbt-template repository`.  Once available, from the root directory of the cloned repo, MacOSX and Linux users may launch the sbt script contained therein.  This will start an SBT session.  Once SBT is started, it is advisable to issue the `console` command; this will start the Scala interpreter.  At this point, you should be able to issue the command `import geotrellis.vector._` without raising an error.
```

Comment:
Confused about `This requires having a running Scala instance with GeoTrellis set as a library dependency`. Does "Scala instance" mean Scala installation? This is actually not necessary.

You go on to recommend installing sbt, but then talk about the sbt script included. If you use the `./sbt` script, you don't have to install SBT.

`Once SBT is started, it is advisable ...` not sure if you're giving advice or giving steps to follow here. One way to make it clearer would be to list out the steps to verify your GeoTrellis clone is working, and bullet point the steps, ending with the import.